.travis.yml

```diff
@@ -13,6 +13,7 @@ before_install:
 - export CHROME_BIN=chromium-browser
 - export DISPLAY=:99.0
 - sh -e /etc/init.d/xvfb start
+- npm install
```

Comment:
I think this is run by default

Reply:
see this one for example: https://travis-ci.org/inspirehep/record-editor/jobs/395351729

Reply:
here too, it ran it only once: https://travis-ci.org/inspirehep/record-editor/jobs/395390501

Maybe they changed it for newer node?

Reply:
actually is using https://docs.npmjs.com/cli/ci which expects ```package-lock.json`` I think

Reply:
hmm... so that package-lock.json should be added to the repo?